File: phpProject/Usuario.php
<?php

class Usuario {
    private $id;
    private $nombre;
    private $apellido;
    private $correo;
    private $contrasena;
    private $fechaNacimiento;
    private $librosDescargados;

    public function __construct($id = null, $nombre = null, $apellido = null, $correo = null, $contrasena = null, $fechaNacimiento = null, $librosDescargados = []) {
        $this->id = $id;
        $this->nombre = $nombre;
        $this->apellido = $apellido;
        $this->correo = $correo;
        $this->contrasena = $contrasena;
        $this->fechaNacimiento = $fechaNacimiento;
        $this->librosDescargados = $librosDescargados;
    }

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getNombre() {
        return $this->nombre;
    }

    public function setNombre($nombre) {
        $this->nombre = $nombre;
    }

    public function getApellido() {
        return $this->apellido;
    }

    public function setApellido($apellido) {
        $this->apellido = $apellido;
    }

    public function getCorreo() {
        return $this->correo;
    }

    public function setCorreo($correo) {
        $this->correo = $correo;
    }

    public function getContrasena() {
        return $this->contrasena;
    }

    public function setContrasena($contrasena) {
        $this->contrasena = $contrasena;
    }

    public function getFechaNacimiento() {
        return $this->fechaNacimiento;
    }

    public function setFechaNacimiento($fechaNacimiento) {
        $this->fechaNacimiento = $fechaNacimiento;
    }

    public function getLibrosDescargados() {
        return $this->librosDescargados;
    }

    public function setLibrosDescargados($librosDescargados) {
        $this->librosDescargados = $librosDescargados;
    }

    public function __toString() {
        return "Usuario{id={$this->id}, nombre='{$this->nombre}', apellido='{$this->apellido}', correo='{$this->correo}', contrasena='{$this->contrasena}', fechaNacimiento={$this->fechaNacimiento}, librosDescargados=" . implode(", ", $this->librosDescargados) . "}";
    }
}

?>

File: javaProyect/co.edu.udec.s2.ejercicio30/src/domain/model/Usuario.java
package domain.model;

import java.time.LocalDate;
import java.util.List;

public class Usuario {
    private int id;
    private String nombre;
    private String apellido;
    private String correo;
    private String contrasena;
    private LocalDate fechaNacimiento;
    private List<Libro> librosDescargados;
    
    public Usuario() {}
    
    public Usuario(int id, String nombre, String apellido, String correo, String contrasena, LocalDate fechaNacimiento, List<Libro> librosDescargados) {
        this.id = id;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;
        this.contrasena = contrasena;
        this.fechaNacimiento = fechaNacimiento;
        this.librosDescargados = librosDescargados;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getContrasena() {
        return contrasena;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }

    public LocalDate getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(LocalDate fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }

    public List<Libro> getLibrosDescargados() {
        return librosDescargados;
    }

    public void setLibrosDescargados(List<Libro> librosDescargados) {
        this.librosDescargados = librosDescargados;
    }

    @Override
    public String toString() {
        return "Usuario{" +
                "id=" + id +
                ", nombre='" + nombre + '\'' +
                ", apellido='" + apellido + '\'' +
                ", correo='" + correo + '\'' +
                ", contrasena='" + contrasena + '\'' +
                ", fechaNacimiento=" + fechaNacimiento +
                ", librosDescargados=" + librosDescargados +
                '}';
    }
}
